documentation/slate/source/includes/_errors.md

```diff
@@ -37,19 +56,28 @@ The are two possible 40x http codes:
 | no_origin_nor_destination   | Couldn't find neither origin nor destination                                | /journeys                 |
 | unknown_object              | Couldn't find one of the request parameters. It can be the region, the API or a PT object | All services              |
 
--   Code 400: bad request
+### 413: request too large
 
-| Error id          | Description                                                 |
-|-------------------|-------------------------------------------------------------|
-| bad_filter        | When you use a custom filter                                |
-| unable_to_parse   | When you use a mal-formed custom filter                     |
-| unknown_api       | When you request an inexistant endpoint                     |
-| bad_format        | When you provide ill-formated parameter (coord for example) |
-| config_exception  | When you try to configure an external service (street network back-end or autocomplete) with wrong parameters |
+When the response generated by the request exceeds 10 MB (few APIs are not yet concerned by this limitation but this may change in the future).
 
+Tips to work around this error:
+
+* [paginate](#paging)
+* reduce the parameter `depth`
+* on large `journeys`, reduce the time frames, durations, _etc_
```

Comment:
`time_frame_duration` is one parameter. Another one of interest would `min_nb_journeys`.

```suggestion
* on large `journeys`, reduce the `time_frame_duration`, `min_nb_journeys`, _etc_
```

Reply:
did you prefer more precision on the workaround? Do you agree the last change?